hw/ip/hmac/dv/cov/hmac_cov_excl.el

```diff
@@ -41,147 +86,218 @@ CHECKSUM: "2796771826 1626244524"
+ANNOTATION: "[UNR] u_rspfifo is not used in tlul_adapter"
 Condition 1 "1864599383" "(gen_normal_fifo.full ? (1'(Depth)) : ((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((1'(gen_normal_fifo.wptr_value) - 1'(gen_normal_fifo.rptr_value))) : (((1'(Depth) - 1'(gen_normal_fifo.rptr_value)) + 1'(gen_normal_fifo.wptr_value))))) 1 -1" (2 "1")
+ANNOTATION: "[UNR] u_rspfifo is not used in tlul_adapter"
 Condition 3 "3985904135" "((gen_normal_fifo.fifo_empty && wvalid) ? wdata : gen_normal_fifo.storage_rdata) 1 -1" (2 "1")
-Condition 2 "3256391329" "((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((1'(gen_normal_fifo.wptr_value) - 1'(gen_normal_fifo.rptr_value))) : (((1'(Depth) - 1'(gen_normal_fifo.rptr_value)) + 1'(gen_normal_fifo.wptr_value)))) 1 -1" (1 "0")
-Condition 4 "1115643068" "(gen_normal_fifo.fifo_empty && wvalid) 1 -1" (3 "11")
-Condition 4 "1115643068" "(gen_normal_fifo.fifo_empty && wvalid) 1 -1" (1 "01")
 CHECKSUM: "2768112785 1802779848"
 INSTANCE: tb.dut.u_tlul_adapter
 ANNOTATION: "[UNSUPPORTED] hmac msg fifo is WO"
 Condition 7 "4228513916" "((((reqfifo_rdata.op == OpRead) & (~reqfifo_rdata.error))) ? reqfifo_rready : 1'b0) 1 -1" (2 "1")
 CHECKSUM: "3518855206 1814878003"
 INSTANCE: tb.dut.u_packer
+ANNOTATION: "[UNR]"
 Condition 2 "2611056762" "(flush_i && ready_i) 1 -1" (2 "10")
 CHECKSUM: "3885231045 4271210983"
 INSTANCE: tb.dut.u_reg.u_socket.err_resp
-ANNOTATION: "[UNR] error resp is handled at tlul interface level"
-Branch 2 "2710626646" "(!rst_ni)" (1) "(!rst_ni) 0,1"
 ANNOTATION: "[UNSUPPORTED] hmac msg fifo is WO"
 Branch 0 "2051317344" "(err_opcode == Get)" (1) "(err_opcode == Get) 0"
-Branch 1 "2688379704" "(!rst_ni)" (3) "(!rst_ni) 0,0,0"
+ANNOTATION: "[UNR] error resp is handled at tlul interface level"
+Branch 2 "2710626646" "(!rst_ni)" (1) "(!rst_ni) 0,1"
 ANNOTATION: "[UNR] error resp is handled at tlul interface level"
 Branch 1 "2688379704" "(!rst_ni)" (1) "(!rst_ni) 0,1,-"
+ANNOTATION: "[UNR]Tool said it is missing else statement, designer waived"
+Branch 1 "2688379704" "(!rst_ni)" (3) "(!rst_ni) 0,0,0"
 CHECKSUM: "2494051891 720836556"
 INSTANCE: tb.dut.u_sha2
-ANNOTATION: "[UNR] FUTURE ITEM: throughput optimization"
-Branch 7 "2566632963" "fifo_st_q" (2) "fifo_st_q FifoLoadFromFifo ,-,1,-,-,-,-"
-Branch 9 "1372145373" "sha_st_q" (6) "sha_st_q ShaUpdateDigest ,-,-,-,1"
-ANNOTATION: "[UNSUPPORTED] FUTURE ITEM: wipe secret"
+ANNOTATION: "[UNSUPPORTED] FUTURE_ITEM: wipe secret"
 Branch 0 "382698320" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-,-,-"
-Branch 1 "3205957143" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "[UNR] FUTURE_ITEM: throughput optimization"
+Branch 9 "1372145373" "sha_st_q" (6) "sha_st_q ShaUpdateDigest ,-,-,-,1"
+ANNOTATION: "[UNR] Sha_en won't be modified during the msg_wr"
```

Comment:
Hi Eunchan @eunchan, 
for the `sha2.sv` code below:
Since sha_en won't be able to be changed during msg_wr, do you still want to keep the `(!sha_en)` branch here?
```
      FifoLoadFromFifo: begin
        if (!sha_en) begin
          fifo_st_d = FifoIdle;
          update_w_from_fifo = 1'b0;
        end else if (!shaf_rvalid) begin
```